content/topics/data-science-specific/google-data-studio/_index.md

```diff
@@ -9,8 +9,6 @@ title: Google Data Studio
 
 Dashboarding is easy with Google Data Studio. Google Data Studio helps you import, manipulate and display your data in an easy to understand way. If you know Google Data Studio, you'll also find other popular data analytics tools such as Power BI (Microsoft), Tableau and Qlikview easy to learn.
 
-The slides for the data visualisation / business intelligence workshop can be found in: {{% contentlink path="workshops/business-intelligence-tools" %}}
```

Comment:
Dont we need to tell people were to find the slides?

Reply:
Fixed